.github/workflows/ci.yml

```diff
@@ -155,10 +155,21 @@ jobs:
         shell: bash
         run: echo ::set-output name=flavor::$([[ "${{ matrix.os }}" == "cuda" ]] && echo "Cuda" || echo "CPU")
 
+      - name: Set values
+        id: values
+        shell: bash
+        run: |
+          if [[ ${{ github.event_name == 'schedule' || (github.event_name == 'push' && (github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/tags/v'))) }} ]]; then
+            jsonValues="{"ILGPU_CLEAN_TESTS":"true", "ILGPU_TEST_PARALLELIZATION_LEVEL":"1"}"
+            echo ::set-output name=json::$jsonValues
+          else
+            jsonValues="{"ILGPU_CLEAN_TESTS":"false", "ILGPU_TEST_PARALLELIZATION_LEVEL":"4"}"
```

Comment:
as these are the default values, I think we could afford to set an empty json object instead (`{}`)